recipes/rpbasicdesign/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "rpbasicdesign" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2c3123bbb2daabba3f7e21ca6896a2bf851b1c2ce1dbb446ab707a198a9853c0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+  run:
+    - python {{ python }}
+    - python-libsbml
+    - pysbol2
+
+test:
+  source_files:
+    - tests
+  imports:
+    - {{ name|lower }}
+  requires:
+    - pytest
+  commands:
+    - python -m {{ name|lower }}.cli --help
+    - cd tests
+    - pytest -v
+
+about:
+  home: https://github.com/brsynth/{{ name|lower }}
+  summary: "Convert rpSBML enzyme info into BASIC construct"
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - tduigou
```

Comment:
Is there a reason why @kenza12 isn't listed here and someone else is instead? (multiple maintainers is fine, but confirmation is needed from the others via a comment or reaction)

Reply:
Hello, I'm OK to appear in the list of maintainers, alongside @kenza12.